src/app/controller/urlVerification.ts

```
@@ -0,0 +1,5 @@
+export const urlVerification = (
+  payload: any,
```

Biome

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)